services/nomad/infrastructure/traefik.nomad

```diff
@@ -87,6 +87,16 @@ job "traefik" {
       service = "vault"
       rule = "Host(`vault.s.voidlinux.org`)"
       [http.routers.vault.tls]
+    [http.routers.repo-ci]
```

Comment:
Hmm, I would prefer this be set as an additional host on the consul service if possible.  Is this not possible to do with the `Host` rule type?